### DISCUSSION TITLE
Unable to access D drive. Also, is there something I'm not understanding about how backups work?

### DISCUSSION THREAD

**Discussion by radiatewishbone** [JonathanTreffler/backblaze-personal-wine-container#143] [General]
See attached conversation I had with Claude AI after copying in the instruction guide from this Github page.

As you can see, I can't access the D drive.

Also, something I feel like I'm not getting is how to properly configure things so that my Unraid NAS is actually sending information to this container to be backed up? Can someone help explain this for a noob.

[Claude AI Backblaze Personal Setup-compressed.pdf](https://github.com/JonathanTreffler/backblaze-personal-wine-container/files/14947208/Claude.AI.Backblaze.Personal.Setup-compressed.pdf)

**Comment by Aractor** [comment]
I notice this about a month ago, but my docker instance is also unable to see the D drive.

I've been trying to troubleshoot by reinstalling the docker image from scratch, and trying different tags, but nothing has made any difference. This previously was working perfectly.

**Comment by Aractor** [comment]
@a-laurence-uk If you're still having this problem, I found the solution in the container's troubleshooting documentation https://github.com/JonathanTreffler/backblaze-personal-wine-container#volumes

Try running the command `docker exec --user app backblaze_personal_backup ln -s /backup_volume/ /config/wine/dosdevices/d:` to manually mount your storage drive as D. This fixed the problem for me.
